package.json

```diff
@@ -51,6 +52,20 @@
     "ts-node": "^8.1.0",
     "typescript": "^3.4.4",
     "@typescript-eslint/eslint-plugin": "^1.7.0",
-    "@typescript-eslint/parser": "^1.7.0"
+    "@typescript-eslint/parser": "^1.7.0",
+    "@commitlint/cli": "^8.2.0",
+    "@commitlint/config-conventional": "^8.2.0",
+    "standard-version": "^7.0.0",
+    "husky": "^3.0.9"
+  },
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
```

Comment:
Husky creates a git hook which enforces correct prefixes of commute messages such as `feat:`, `fix:` or `test:`